release/_releases_json.sh

```diff
@@ -52,7 +52,19 @@ function _process_product {
 		tr -d "/" | \
 		uniq)
 	do
-		_process_product_version "${product_name}" "${product_version}"
+		local product_version_trivial=$(echo "${product_version}" | grep -oE '(\.u|-u)[0-9]+' | grep -oE '[0-9]+')
+
+		if [[ "${product_version}" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ && "${product_version:2:1}" = 4 ]]
```

Comment:
@pgydev, let's talk about this pull by next Tuesday. Please reach me out through Slack. This latest version is starting to make it more complex and it may be refused by BChan as the first version of this fix.

Reply:
@natocesarrego this regex's purpose was to see if the release's format is 7.x.x or 2024.qx.x, but i relaized that i don't have to check it.